scripts/frontends/onnx-frontend/build_and_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I think moving `scripts/frontends/onnx-frontend/` to `frontends/onnx-frontend/scripts/` is more suitable. It will make your yaml more simple.

Reply:
I will move torch-frontend's script later.

Reply:
Do we need to move the scripts of compiler and runtime as well?

Reply:
> Do we need to move the scripts of compiler and runtime as well?

I think not, compiler and runtime are binding to the same llvm version. They should run and test together.